tests/cases/conformance/jsdoc/jsdocTypesInTypeAnnotations.ts

```diff
@@ -0,0 +1,22 @@
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+// @module: commonjs
+// @filename: node.d.ts
+// @noImplicitAny: true
+declare function require(id: string): any;
```

Comment:
This doesn't seem relevant to the test?
Also, the test name seems to imply that jsdoc types will appear inside of type annotations.